README.md

```diff
@@ -126,6 +126,7 @@ type Profile = {
 - `additionalParams`: dictionary of additional query params to add to all requests; if an object with this key is passed to `authenticate`, the dictionary of additional query params will be appended to those present on the returned URL, overriding any specified by initialization options' additional parameters (`additionalParams`, `additionalAuthorizeParams`, and `additionalLogoutParams`)
 - `additionalAuthorizeParams`: dictionary of additional query params to add to 'authorize' requests
 - `identifierFormat`: optional name identifier format to request from identity provider (default: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`)
+- `wantAssertionsSigned`: if truthy, add `WantAssertionsSigned="true"` to the metadata, to specify that the IdP should always sign the assertions.
```

Comment:
Actually, "truthy" isn't the correct thing to say here. The `??` will test for ` == null`, which is `null` and `undefined`. We might want to change the ctor to coerce this value to boolean using `!!` if that is what we really want.

Reply:
If we use `!!`, that would be the odd one out within all the other optional parameters that use `?? false`.

A list of all options using `?? false` and the wording in the readme:
* passive: **not found in readme**
* disableRequestedAuthnContext: "if truthy"
* forceAuthn: "if set to true"
* skipRequestCompression: "if set to true"
* disableRequestAcsUrl: "if truthy"
* wantAssertionsSigned: "if truthy"
* validateInResponseTo: "if truthy"

By using the phrase "if truthy" I follow the same structure as most of the options.

I would prefer to use the same wording everywhere, and to me "if set to true" sounds more professional and has my preference, but than we should change all these options in the readme.

These changes in the readme should probably be done in another PR.
Probably combined with alphabetically ordering the readme and the code, and adding `passive` to the readme.